src/core/fetchIntroSteps.js

```diff
@@ -26,14 +26,37 @@ export default function fetchIntroSteps(targetElm) {
       //use querySelector function only when developer used CSS selector
       if (typeof currentItem.element === "string") {
         //grab the element with given selector from the page
-        currentItem.element = document.querySelector(currentItem.element);
-      }
-
-      //intro without element
-      if (
+        const el = document.querySelector(currentItem.element);
+        if (el !== null) {
+          currentItem.element = el;
+        } else {
+          // If element is not exists yet, we'll get it on step
+          const elSelector = currentItem.element;
+          Object.defineProperty(currentItem, "element", {
```

Comment:
Is it important to have this object when the DOMElement doesn't exist? Can we just replace it with `undefined`?

Reply:
Yes, my first (local) implementation was with `undefined`, but it could be more changes in other code, that calls `.element`. 

I'll try to do with `undefined`.